integration/spark/src/main/scala/org/apache/spark/sql/parser/CarbonSpark2SqlParser.scala

```diff
@@ -377,7 +399,12 @@ class CarbonSpark2SqlParser extends CarbonDDLSqlParser {
 
   protected lazy val element: Parser[String] =
     (ident <~ ".").? ~ ident ^^ {
-      case table ~ column => column.toLowerCase
+      case table ~ column =>
+        if (table.isDefined) {
+          table.get + "." + column.toLowerCase
```

Comment:
No need to convert `table.get` to lowercase ?

Reply:
Done